### ISSUE TITLE
Vpn Start Failure

### ISSUE THREAD

**Issue by suki-neithnet** [Jigsaw-Code/outline-client#1828]
**Describe the issue**
I've already follow the tips from https://github.com/Jigsaw-Code/outline-client/tree/master/src/cordova/apple#debug-vpnstartfailure, but still not working, sometimes it connect on but will disconnect then reconnect in short seconds for several times. To the end, although the connection time is running, but actually it still not connect to server.

**Environment**
 - OS: macOS Sonoma 14.2.1
 - NPM version 10.2.3
 - Node.js version 18.19

**Comment by sbruens** [comment]
We don't fully support maccatalyst yet, for this exact reason. This happens only on M1 machines as far as we have been able to see. Can you confirm you are on a Silicone M1 machine?

This issue is blocking our migration from macos->maccatalyst to resolve #1727, but we don't have a specific issue for this bug so thank you for starting one.

**Comment by suki-neithnet** [comment]
> We don't fully support maccatalyst yet, for this exact reason. This happens only on M1 machines as far as we have been able to see. Can you confirm you are on a Silicone M1 machine?
> 
> This issue is blocking our migration from macos->maccatalyst to resolve #1727, but we don't have a specific issue for this bug so thank you for starting one.

Yes, I'm on a Silicone M1 machine, so that's the problem ! OK, thank you very much.

**Comment by PL7963** [comment]
Same with Macbook Air M2.
